scipy/stats/_survival.py

```diff
@@ -15,26 +15,53 @@ class EmpiricalDistributionFunction:
 
     Attributes
     ----------
-    points : ndarray
-        The point estimate of the cumulative distribution function (CDF) or its
-        complement, the survival function (SF), at unique values of the sample.
+    q : ndarray
+        The unique values of the sample from which the
+        `EmpiricalDistributionFunction` was estimated.
+    p : ndarray
+        The point estimates of the cumulative distribution function (CDF) or
+        its complement, the survival function (SF), corresponding with `q`.
```

Comment:
For private attributes I don't really mind short names but I would prefer if we could use full names for public things. `quantile` and `points` (or `estimate`) would be clearer to me.

Reply:
I can go for `quantiles`. I think the plural `s` is important to emphasize that this is several discrete points - the number of unique points in the sample.

Then the question would be what is the name of the output if "quantiles" is the input. "points" and "estimate" don't go particularly well with "quantiles". "percentiles" sounds nice, but I think `probabilities` is the most descriptive.

LMK if you want it changed to `quantiles` and `probabilities`.

Reply:
I would go for `quantiles` and `probabilities` (totally agree for the plural, my mistake above!) Thanks 😃

Reply:
OK, although by the same logic we'd write out `cumulative_distribution_function` and `survival_function` : /